src/naming/Naming_AST.ml

```diff
@@ -471,6 +471,45 @@ let assign_implicitly_declares lang =
   lang =*= Lang.Python || lang =*= Lang.Ruby || lang =*= Lang.Php
   || Lang.is_js lang
 
+(*****************************************************************************)
```

Comment:
I don't think this should go into `Naming`, they're unrelated. You can just add a new module in `analyzing/` that performs this step, and then perhaps call it from `Parse_target` when we already compute the CFG to perform const-prop.

Reply:
Sounds good.

Reply:
Moved things to a different file as suggested.